src/logic/unique.lean

```diff
@@ -57,7 +57,10 @@ instance punit.unique : unique punit.{u} :=
 { default := punit.star,
   uniq := λ x, punit_eq x _ }
 
-instance : unique true := { default := trivial, uniq := λ x, rfl }
+def unique_prop {p : Prop} (h : p) : unique p :=
+{ default := h, uniq := λ x, rfl }
```

Comment:
is the `λ x` stuff preferred over `@rfl p`?

Reply:
I've never seen `@rfl` used.